js/main.js

```diff
@@ -23,6 +30,165 @@ function addCats(eachCat) {
+
+  var infoValues = document.createElement('div');
+  infoValues.className = 'col-half half';
+
+  var weightCol = document.createElement('div');
+  weightCol.className = 'col-half';
+
+  var weightName = document.createElement('h2');
+  weightName.textContent = 'Weight';
+
+  var weightValue = document.createElement('p');
+  weightValue.textContent = eachCat.weight.imperial + ' pounds';
+  weightValue.className = 'info-value';
+
+  weightCol.appendChild(weightName);
+  weightCol.appendChild(weightValue);
+  infoValues.appendChild(weightCol);
+
+  var lifeSpanCol = document.createElement('div');
+  lifeSpanCol.className = 'col-half';
+
+  var lifeSpanName = document.createElement('h2');
+  lifeSpanName.textContent = 'Life Span';
+
+  var lifeSpanValue = document.createElement('p');
+  lifeSpanValue.textContent = eachCat.life_span + ' years';
+  lifeSpanValue.className = 'info-value';
+
+  lifeSpanCol.appendChild(lifeSpanName);
+  lifeSpanCol.appendChild(lifeSpanValue);
+  infoValues.appendChild(lifeSpanCol);
+
+  var energyLevelCol = document.createElement('div');
+  energyLevelCol.className = 'col-half';
+
+  var energyLevelName = document.createElement('h2');
+  energyLevelName.textContent = 'Energy Level';
+
+  var energyLevelValue = document.createElement('div');
+  energyLevelValue.className = 'paws';
+  for (var i = eachCat.energy_level; i > 0; i--) {
+    var paw = document.createElement('i');
+    paw.className = 'fas fa-paw';
+    energyLevelValue.appendChild(paw);
+  }
+
+  energyLevelCol.appendChild(energyLevelName);
+  energyLevelCol.appendChild(energyLevelValue);
+  infoValues.appendChild(energyLevelCol);
+
+  var affectionCol = document.createElement('div');
+  affectionCol.className = 'col-half';
+
+  var affectionName = document.createElement('h2');
+  affectionName.textContent = 'Affection';
+
+  var affectionValue = document.createElement('div');
+  affectionValue.className = 'paws';
+  for (i = eachCat.affection_level; i > 0; i--) {
+    paw = document.createElement('i');
+    paw.className = 'fas fa-paw';
+    affectionValue.appendChild(paw);
+  }
+
+  affectionCol.appendChild(affectionName);
+  affectionCol.appendChild(affectionValue);
+  infoValues.appendChild(affectionCol);
+
+  var childFriendlyCol = document.createElement('div');
+  childFriendlyCol.className = 'col-full';
+
+  var childFriendlyName = document.createElement('h2');
+  childFriendlyName.textContent = 'Child Friendly';
+
+  var childFriendlyValue = document.createElement('div');
+  childFriendlyValue.className = 'paws';
+  for (i = eachCat.child_friendly; i > 0; i--) {
+    paw = document.createElement('i');
+    paw.className = 'fas fa-paw';
+    childFriendlyValue.appendChild(paw);
+  }
+
+  childFriendlyCol.appendChild(childFriendlyName);
+  childFriendlyCol.appendChild(childFriendlyValue);
+  infoValues.appendChild(childFriendlyCol);
+  infoRow.appendChild(infoValues);
+  infoViewContainer.appendChild(infoRow);
+
+  return infoViewContainer;
+}
```

Comment:
Per my other comment, once you do that, this can be split into two functions, one named `renderCatListItem` and one named `renderCatDetailView